### ISSUE TITLE
A traveltime bug about the gradient model

### ISSUE THREAD

**Issue by geohang** [gimli-org/gimli#323]
### Problem description

I try to generate a gradient velocity model from my field scene. It is very weird that when I try to generate the 200m/s to 1000m/s from top to bottom. The actual output is a 1000m/s to 200m/s velocity model. When I do an inversion to my field data, the result shows that the inverted model has low velocity in the bottom and we all know that it is irrealistic. 

### Your environment

**Operating system**: Windows
**Python version**: 3.7
**pyGIMLi version**: 1.1.1+5.gc107727a
**Way of installation**:  Conda package,

### Steps to reproduce
```
import pygimli as pg
from pygimli.physics import TravelTimeManager
import pygimli.physics.traveltime as tt
data = tt.load("srtfieldline1.dat")
mgr = TravelTimeManager(data)
mesh=mgr.createMesh(data,quality=32)
sm = tt.createGradientModel2D(data,mesh, vTop=200,vBot=1000)
pg.show(mesh, data=1/sm, label=pg.unit('vel'), showMesh=True)
```

### Expected behavior

generate the 200m/s to 1000m/s velocity model.

### Actual behavior

generate the 1000m/s to 200m/s velocity model.

The field data is attached to this google drive.
https://drive.google.com/file/d/1FVHmYpaHjWml6gxid0yKwbXLh--umtAC/view?usp=sharing

Thanks
Hang

**Comment by halbmy** [comment]
For the gradient model, obviously the `y` coordinate is taken which does not make sense. So if you exchange `y` and `z` in the token line, all will be well. I fixed this in dev@5b973c08. Thanks for reporting.

**Comment by geohang** [comment]
> For the gradient model, obviously the `y` coordinate is taken which does not make sense. So if you exchange `y` and `z` in the token line, all will be well. I fixed this in dev@5b973c08. Thanks for reporting.

Thank you very much!
